yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -828,11 +828,7 @@ func (i *IndexIssueDetector) reportVariousIndexPerfOptimizationsOnFirstColumnOfI
 
 	//Precendence here is if the index has low-cardinality issue then most frequent value issue is not relevant as the user will have to fix the low cardinality index
```

Comment:
can you pls fix the comments around low-cardinality here in this whole function? will be confusing in future